dffml/service/dev.py

```diff
@@ -339,25 +352,34 @@ async def run(self):
             raise RuntimeError("pip failed to install dependencies")
 
 
+@configdataclass
+class SetupPyKWArgConfig:
+    kwarg: str = field(
+        "Keyword argument to write to stdout", position=0,
+    )
+    setupfilepath: str = field(
+        "Path to setup.py", position=1,
+    )
+
+
 class SetupPyKWArg(CMD):
     """
     Get a keyword argument from a call to setup in a setup.py file.
     """
 
-    arg_kwarg = Arg("kwarg", help="Keyword argument to write to stdout")
-    arg_setup_filepath = Arg("setup_filepath", help="Path to setup.py")
+    CONFIG = SetupPyKWArgConfig
 
     @staticmethod
-    def get_kwargs(setup_filepath: str):
-        setup_filepath = Path(setup_filepath)
+    def get_kwargs(setupfilepath: str):
```

Comment:
`setup_filepath` got converted to `setup-filepath`. Removing the underscore fixed the error for now.

Reply:
Sounds good